cli/src/opts/cast.rs

```diff
@@ -333,8 +333,8 @@ Examples:
     #[clap(visible_alias = "t")]
     #[clap(about = "Get information about a transaction.")]
     Tx {
-        #[clap(value_name = "HASH")]
-        hash: String,
+        #[clap(value_name = "TX_HASH")]
+        tx_hash: String,
```

Comment:
this needs `long = "hash"` because it was renamed?

Reply:
does it? it's a positional arg